lib/app/widgets/banners.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:banners/banners.dart';
+import 'package:flutter/material.dart';
+import 'package:schulcloud/app/app.dart';
+
+class OfflineBanner extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return BannerScaffold(
+      backgroundColor: Theme.of(context).errorColor.withOpacity(0.5),
+      body: Padding(
+        padding: EdgeInsets.all(8),
```

Comment:
I'd prefer a horizontal padding of 16 so it matches our globally used horizontal padding (i.e., that of the AppBar title and list items).
(Also applies to the `TokenExpiredBanner` below.)